pkg/kubelet/status/manager.go

```diff
@@ -130,6 +136,8 @@ func (m *manager) Start() {
 		select {
 		case syncRequest := <-m.podStatusChannel:
 			m.syncPod(syncRequest.podUID, syncRequest.status)
+		case podToReconcile := <-m.podReconcileChannel:
```

Comment:
nit: s/podToReconcile/pod

It's quite clear that this is a pod to reconcile since it comes from the channel :)